conda/exceptions.py

```diff
@@ -486,6 +481,36 @@ def __init__(self, message):
         super(CondaVerificationError, self).__init__(message)
 
 
+class CondaDependencyError(CondaError):
+    def __init__(self, message):
+        super(CondaDependencyError, self).__init__(message)
+
+
+class PrefixReplacementError(CondaError):
+    def __init__(self, message, new_data_length, original_data_length, new_prefix,
+                 path, placeholder):
+        message = dals("""
+        Refusing to replace data of length '%(new_data_length)d' with
+        data of length '%(original_data_length)d' for binary file
+        New Data Length:      %(new_data_length)d
```

Comment:
Lower case "new data length" and indent two spaces.  Only use one space after colon.  Make formatting of parameters below similar.